src/js/components/DateInput/DateInput.js

```diff
@@ -111,6 +112,13 @@ Use the icon prop instead.`,
       );
     }
 
+    const reverse = reverseProp || restOfInputProps.reverse;
+
+    const calendarDropdownAlign = {
+      top: 'bottom',
+      ...(reverse ? { right: 'right' } : { left: 'left' }),
+    };
```

Comment:
I think the alignment of the calendar should stay the same even if we use the reverse prop. The reverse prop should only affect the icon.

Reply:
If I don't do this the Input will be this way:

![image](https://user-images.githubusercontent.com/23138717/172375325-619211a2-eef3-45cc-9e16-c04d40c2ea22.png)

Would that be the expected behavior?

Reply:
Very nice tip